README.md

```diff
@@ -1,37 +0,0 @@
-# Theatria
```

Comment:
Can you do me a favor and put this README.md file back in? It's for the github repo and separate from the docs.